src/cmakeProject.ts

```diff
@@ -1963,7 +2156,29 @@ export class CMakeProject {
 
     async ctest(): Promise<number> {
         const drv = await this.preTest();
-        return (await this.cTestController.runCTest(drv)) ? 0 : -1;
+        const retc = await this.cTestController.runCTest(drv);
+        return (retc) ? 0 : -1;
+    }
+
+    async cpack(): Promise<number> {
+        this.isBusy.set(true);
+        const drv = await this.preTest();
+        const retc = await this.cPackageController.runCPack(drv);
```

Comment:
Same comment here, why aren't we just returning `retc`? I think doing this would break the integrity of what cpack actually returned.

Reply:
You are right. Do you think we should change for ctest as well?

Reply:
If you can confirm that it still works as expected with that change, I'm fine with it. However, noticing now that it was like this before your changes, so for safety reasons, I'm okay leaving it.